drivers/overlay/composefs_supported.go

```diff
@@ -75,7 +75,13 @@ func enableVerityRecursive(path string) error {
 	return filepath.WalkDir(path, walkFn)
 }
 
-func generateComposeFsBlob(toc []byte, destFile string) error {
+func getErofsBlob(dataDir string) string {
+	return filepath.Join(dataDir, "erofs.blob")
```

Comment:
Maybe we can name it something with composefs or cfs, as it is not any old erofs image.

Reply:
Hmm, i guess that is in the directory name

Reply:
renamed to `composefs.blob`